app/_src/gateway/kong-enterprise/signing-verify/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Verify signatures for Signed Kong Images
+badge: enterprise
+---
+
+Starting with {{site.ee_product_name}} 3.5.0.2, Docker container images are now signed using `cosign` with signatures published to a Docker Hub repository.
+
+This guide provides steps to verify signatures for signed {{site.ee_product_name}} Docker container images in 2 different ways. The first is the minimal example, used to verify an image without leveraging any annotations. The second is the complete example, leveraging optional annotations for increased trust.
+
+For the minimal example, you only need: Docker details, Github repo name, and Github workflow filename.
+For the complete example, you will need the same details from above, as well as any of the optional annotations you wish to verify:
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | Github repository | `kong-ee` |
+| `<workflow filename>` | Github workflow filename | `release.yml` |
+| `<workflow name>` | Github workflow name | `Package & Release` |
```

Comment:
These feel brittle to be using as build provenance annotations. Is the CA identity regex + repo name not enough?

Reply:
Correct:

```bash
$ cosign verify kong/kong-gateway:3.5.0.2-ubuntu@sha256:208c23d88fa9f563a907dbb29c629c10c6183637898b34cbbafef779ff0965f2 --certificate-oidc-issuer='https://token.actions.githubusercontent.com' | jq

Error: --certificate-identity or --certificate-identity-regexp is required for verification in keyless mode
main.go:74: error during command execution: --certificate-identity or --certificate-identity-regexp is required for verification in keyless mode
```

Reply:
> For the minimal example, you only need: Docker details, Github repo name, and Github workflow filename.

As in, the minimum required annotations for the command to succeed.